### ISSUE TITLE
Integrated terminal starts cutting off first letter

### ISSUE THREAD

**Issue by garfieldbanks** [Microsoft/vscode#11927]
- VSCode Version:152
- OS Version:10Ann

Steps to Reproduce:
1. Use integrated terminal
2. Use git command
3. Interrupt git command with ctrl+c
4. Try a git command
5. Read message _'it' is not recognized as an internal or external command, operable program or batch file._

**Comment by Tyriar** [comment]
Can you provide a screenshot of the issue?

**Comment by garfieldbanks** [comment]
I will next time it happens.

Are there any other logs I can send that will be helpful?

**Comment by Tyriar** [comment]
If it happens again, try pressing enter a few times and type the command again. Also the output of `echo %PATH%` would be useful.

**Comment by garfieldbanks** [comment]
C:\WINDOWS;
C:\WINDOWS\system32;
C:\WINDOWS\System32\Wbem;
C:\WINDOWS\System32\WindowsPowerShell\v1.0\;
C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\;
C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;
C:\Program Files\Anaconda3;
C:\Program Files\Anaconda3\Scripts;
C:\Program Files\Anaconda3\Library\bin;
C:\Program Files (x86)\GtkSharp\2.12\bin;
C:\Program Files\Microsoft DNX\Dnvm\;
C:\Users\Garfield\AppData\Local\Microsoft\WindowsApps;
C:\Users\Garfield\Apps\Git\cmd;
C:\Users\Garfield\Apps\Git\mingw64\bin;
C:\Users\Garfield\Apps\Git\usr\bin;
C:\Users\Garfield\Apps\VisualStudioCode\bin;
C:\Users\Garfield\Apps\Node;
C:\Users\Garfield\Apps\Python35\Scripts\;
C:\Users\Garfield\Apps\Python35\;
C:\Program Files\Java\jdk1.8.0_102\bin
C:\ProgramData\Oracle\Java\javapath;

**Comment by garfieldbanks** [comment]
Happened again. Here's the screenshot:

![it 1](https://cloud.githubusercontent.com/assets/12904270/18546359/c55f5956-7afa-11e6-9769-350fd7275014.PNG)

There are no spaces in the path, if that is at all important.

I just tried git status after taking that screenshot and uploading it and it worked. It was the next command after the 'cho' is not recognized... error.

Using press the up arrow puts the cursor on the line above where it should be. It shows that my last commands were 'cho %PATH%' and 'it status'.

When I press the down arrow it now puts me on the row below where it should be and displays the 'git status' command that worked after uploading the screenshot.

Path is still the same as it was in my comment above.

So yeah I didn't have to press enter at all. It just worked after I came back from creating/uploading the screenshot.

Anything else I should do while I have this terminal open?
